core/templates/pages/about-page/about-page.component.spec.ts

```diff
@@ -16,161 +16,209 @@
  * @fileoverview Unit tests for the about page.
  */
 
+import { ComponentFixture, fakeAsync, TestBed} from '@angular/core/testing';
+import { EventEmitter, NO_ERRORS_SCHEMA, Pipe, PipeTransform }
+  from '@angular/core';
+
+import { AboutPageComponent } from './about-page.component';
+import { I18nLanguageCodeService } from 'services/i18n-language-code.service';
+import { TranslateService } from 'services/translate.service';
+import { UrlInterpolationService } from
+  'domain/utilities/url-interpolation.service';
+import { UtilsService } from 'services/utils.service';
 import { WindowRef } from 'services/contextual/window-ref.service';
 
-require('pages/about-page/about-page.component.ts');
+@Pipe({name: 'translate'})
+class MockTranslatePipe {
+  transform(value: string, params: Object | undefined):string {
+    return value;
+  }
+}
+
+class MockTranslateService {
+  code = 'es';
```

Comment:
Here and elsewhere, please use more descriptive names. In this case, languageCode. And perhaps in use() below: newLanguageCode.

Otherwise, the reader has no idea what is going on. When I first saw "code" I thought it referred to a string of e.g. JS/Python code. Make sure your variable names are descriptive and that they convey meaning.

Reply:
Changed!